pom.xml

```diff
@@ -183,11 +183,13 @@
         <plantuml.version>6121</plantuml.version>
 
         <!-- Test dependencies -->
+        <testng.version>6.10</testng.version>
```

Comment:
`testng.version` and `mockito.version` now appear twice in the file - remove this.